src/universal/bin/mantis-launcher

```diff
@@ -3,11 +3,10 @@
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 cd $DIR/..
 
-chain="$1"
-if [ -z "$chain" ]
-then
-  echo "You need to choose chain"
-else
+CONFIG_FILE="./conf/$1.conf"
+if [ -f "$CONFIG_FILE" ]; then
   shift
-  ./bin/mantis -Dconfig.file=./conf/"$chain".conf "$@"
+  CHAIN_PARAM="-Dconfig.file=$CONFIG_FILE"
 fi
+
+./bin/mantis ${CHAIN_PARAM:+"$CHAIN_PARAM"} "$@"
```

Comment:
What does this fallback from `CHAIN_PARAM` to itself achieve?

Reply:
I found this is the way in bash to not add anything if `CHAIN_PARAM` is not set. Otherwise it adds the empty string and makes the command fail